controller/model/associations_handlers.go

```diff
@@ -59,8 +59,10 @@ func (handler *AssociationsHandler) UpdateAssociations(store persistence.Store,
 			return linkCollection.AddLinks(tx, parentId, childIds...)
 		case AssociationsActionRemove:
 			return linkCollection.RemoveLinks(tx, parentId, childIds...)
-		default:
+		case AssociationsActionSet:
```

Comment:
Wasn't mentioned in the commit message. Was this on purpose?

Reply:
oh, yes. Small tidy. Make it explicit which actions are supported in case we ever add new ones